app/src/main/java/ch/epfl/qedit/view/quiz/QuestionFragment.java

```diff
@@ -50,9 +57,14 @@ public void onChanged(Integer index) {
 
     /** Handles the transition from one question to another */
     private void onQuestionChanged(Quiz quiz, Integer index) {
-        if (index == null || quiz == null || index < 0 || index >= quiz.getQuestions().size())
-            return;
+        System.out.println("Heanvounsoinrvoirnv ori novi iorn eoin voinr oiren ");
+        System.out.println(quiz);
+        System.out.println(index);
 
+        if (index == null || quiz == null || index < 0 || index >= quiz.getQuestions().size()) {
+            return;
+        }
+        System.out.println("Question question = quiz.getQuestions().get(index);");
```

Comment:
yes, same comment as above, just highlighting (it is not spelled correctly right ^^ ) so that you don't forget to delete it.